tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Get Organisation Users.5147.feature

```diff
@@ -8,40 +8,50 @@ Background:
         | Name           | OdsCode |
         | Organisation 1 | Ods 1   |
     And Users exist
-        | Id  | OrganisationName | FirstName | LastName | Email   | PhoneNumber | Disabled | Password |
-        | 123 | Organisation 1   | John      | Doe      | a@b.com | 01234567890 | false    | yolo     |
+        | Id  | OrganisationName | FirstName | LastName | Email             | PhoneNumber | Disabled | Password | OrganisationFunction |
+        | 123 | Organisation 1   | John      | Doe      | authority@doe.com | 01234567890 | false    | yolo     | Authority            |
+        | 234 | Organisation 1   | Jane      | Doe      | buyer@doe.com     | 01234567890 | false    | oloy     | Buyer                |
 
 @5147
 Scenario: 1. Get all of the users for an organisation
     Given an user is logged in
```

Comment:
Given a authority user is logged in

Reply:
That step doesn't exist anymore, right? I'm logging in as authority@doe.com which logs us in as an authority